features/camel/src/main/feature/feature.xml

```diff
@@ -27,7 +27,7 @@
     <!--START Camel Features-->
     <!-- DO NOT EDIT
         All feature definitions come from 3rd party feature file:
-        <repository>mvn:org.apache.camel.karaf/apache-camel/2.22.0/xml/features</repository>
+        <repository>mvn:org.apache.camel.karaf/apache-camel/2.24.2/xml/features</repository>
```

Comment:
✏️ It looks like camel updated the netty version that they use to `4.1.32.Final`, so we should probably update the version here
https://github.com/codice/ddf/blob/a755c443eccb3696ab62b09f3e906c51402b355d/pom.xml#L258

Reply:
Can do. I had that upgrade on our sec ticket backlog but I can do it here instead 👍